src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -158,6 +159,20 @@ public class DurableTaskExtension :
         /// </value>
         public bool DisableHttpManagementApis { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value of Event Grid Topic Endpoint for emitting lifecycle events.
+        /// If this property has been set, it will emit the event to the Event Grid Topic.
+        /// </summary>
+        public string EventGridTopicEndpoint { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value of Event Grid Key.
+        /// If this property has been set, it will emit the event to the Event Grid Topic.

```

Comment:
I would reword the summary like this:
*Gets or sets the name of the app setting containing the key used for authenticating with the Azure Event Grid custom topic at `<see cref="EventGridTopicEndpoint"/>.`*

Reply:
Great! We will change it. Thank you!